### PR TITLE
Feature/LP 32 repair request UI

### PR DESCRIPTION
**Pull request by JoseGR98**
LP-32 Repair Request UI
https://devaptivist.atlassian.net/browse/LP-32
Repair User Interface. View mainly composed of text fields to fill a form, to help the client with problems in his/her apartment.
3 files added & edited:
- RepairView
- Gradle of RepairView
- Strings of RepairView

RepairUI

### CODE REVIEW

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringArrayResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.iteneum.designsystem.components.*
+import com.iteneum.designsystem.theme.LPTypography
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.repair.R
+
```
Comment (melissaquezada94) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt:14]: Please add the documentation for this function with the `@author`

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+@OptIn(ExperimentalMaterial3Api::class)
+@Preview
+@Composable
+fun RepairView() {
+    val dp24 = LeasePertTheme.sizes.regularSize
```
Comment (melissaquezada94) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: I recommend you use just one variable for the sizes container `val sizes = LeasePertTheme.sizes` and then use the sizes in your code to get all the ones that you need. For example if you need different extraSizes and also the midSmall size you use do the following:
`sizes. extraSize124`
`sizes. extraSize128`
`sizes. midSmallSize`
`sizes. minorRegularSize`
it is just to simplify the code and not create an each variable for each size

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    val dp20 = LeasePertTheme.sizes.minorRegularSize
+    val eSdp06 = LeasePertTheme.sizes.extraSize6
+    val eSdp10 = LeasePertTheme.sizes.extraSize10
+    val eSdp104 = LeasePertTheme.sizes.extraSize104
+    val arrayRadioButtons = stringArrayResource(id = R.array.list_RadioButton)
```
Comment (melissaquezada94) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: Could be more descriptive with the name of variables, please? Something like permissionRadioButtons

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    val arrayPetInUnit = stringArrayResource(id = R.array.list_PetInUnit)
+    val arrayCategory = stringArrayResource(id = R.array.list_Category)
+    var isValidDescription by remember {
+        mutableStateOf(false)
+    }
+    var selectedOptionRadioButtons by remember {
```
Comment (melissaquezada94) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: In programming, is common to see the selected at the end like: `optionSelectedRadioButtons`

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+            LPPhoneNumberText(
+                modifier = Modifier.padding(
+                    top = eSdp10
+                ),
+                value = numberValue,
+                onPhoneChange = {
+                    numberValue = it
+                }
+            )
+            /*LpOutlinedTextFieldNumber(
```
Comment (melissaquezada94) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: Please delete the commented code

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+                ),
+                style = LPTypography.bodyLarge
+            )
+            LpFileButton(
+                modifier = Modifier
+                    .padding(
+                        top = eSdp06
+                    )
+                    .fillMaxWidth(),
+                mimeTypes = arrayOf("Option A", "Option B"),
```
Comment (melissaquezada94) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: meanwhile you can left this mime types just with `"video/*"`

File: feature/repair/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+<resources>
+
+    <string name="text_title">Repair request</string>
```
Comment (melissaquezada94) [inline feature/repair/src/main/res/values/strings.xml:3]: Please check the nomenclature file for the strings

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+                label = stringResource(id = R.string.label_Description),
+                hint = stringResource(id = R.string.hint_Description),
+                isValid = isValidDescription,
+                supportTextError = stringResource(id = R.string.supportError_Description),
+                onValueChange = {
+                    isValidDescription = it.isEmpty() || it.matches(Regex(".*[a-zA-Z]+.*")).not()
+                }
+            )
+            Text(
+                text = stringResource(id = R.string.text_Video),
```
Comment (carlosph816) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: if you are using snake case this string should be "text_video"

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+                style = LPTypography.bodyLarge
+            )
+            LpFileButton(
+                modifier = Modifier
+                    .padding(
+                        top = eSdp06
+                    )
+                    .fillMaxWidth(),
+                mimeTypes = arrayOf("Option A", "Option B"),
+                onFileSelected = {}
```
Comment (carlosph816) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt:129]: Can you add a "TODO" to know how it's going to work

Comment (JoseGR98) [reply feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt:129]: Corrections were made. Thank you @carlosph816.

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt (hunk 6 above)
Comment (JoseGR98) [reply feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: Corrections were made. Thank you @melissaquezada94

File: feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringArrayResource
+import androidx.compose.ui.res.stringResource
+import com.iteneum.designsystem.components.*
+import com.iteneum.designsystem.theme.LPTypography
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.repair.R
+
+/**
+ * Function that creates [RepairView] compose. for user's service request
```
Comment (melissaquezada94) [inline feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: Please follow the documentation:
- Description
- params
- retunrs
- author
If it doesn't have return and params just add the description and author please

Comment (JoseGR98) [reply feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt]: Ok, I erased the other values and focused on description only :)

**Review by carlosph816 — APPROVED** [review]
Well done!

**Review by melissaquezada94 — APPROVED** [review]

**Pull request merged**